### ISSUE TITLE
[Rare mobs] drops are white

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Blackwing-Lair#308]
Rare mobs drops are white items.
Aren't Rare mobs meant to drop good stuff and give extra XP ?
this is my 4rd rare and crappy drops and same XP and a normal mob
https://www.wowhead.com/npc=47012/effritus#drops:mode=
![rare effritus](https://user-images.githubusercontent.com/39439201/50549389-6014a100-0c5c-11e9-90ad-247e3fb39d0d.jpg)

https://www.wowhead.com/npc=47009/aquarius-the-unbound
![rare aquarius the unbound](https://user-images.githubusercontent.com/39439201/50549391-6276fb00-0c5c-11e9-8b3c-9c6c3bdcef80.jpg)

https://www.wowhead.com/npc=12431/gorefang
![rare gorefang](https://user-images.githubusercontent.com/39439201/50549392-6571eb80-0c5c-11e9-80a5-fced1a44edf6.jpg)

https://www.wowhead.com/npc=14223/cranky-benj
![cranky ben](https://user-images.githubusercontent.com/39439201/50552229-ffeb2280-0c8e-11e9-8145-cc2bbcaf088a.jpg)

![crappy rewards and no extra xp](https://user-images.githubusercontent.com/39439201/50549415-beda1a80-0c5c-11e9-9838-b58d4119ea46.jpg)

![crappy rewards](https://user-images.githubusercontent.com/39439201/50549421-eb8e3200-0c5c-11e9-9fff-1fbc707f603d.jpg)

**Comment by Echinaceax** [comment]
Ok i will link wowhead info but here is one ive found and if you look at the rewards for Aquarious the unbound they are several greens 

https://www.wowhead.com/npc=47009/aquarius-the-unbound

**Comment by Echinaceax** [comment]
"I think that they do give extra xp that is scaled with level I believe"

i recently lvled up ...ive gone back to test it again 
they give the same XP (as i said before) :)

![same xp](https://user-images.githubusercontent.com/39439201/50549899-a3274200-0c65-11e9-8461-3eb41256dd23.jpg)

**Comment by Echinaceax** [comment]
Same with cranky benj (added to first post)

**Comment by Dranah** [comment]
This is the same on wotlk/redemption in vanilla zones. Most old-zone rares drop trash loot.

Here's description from WoWpedia
"The loot dropped from rare mobs is unique and located almost exclusively within the uncommon table and below it (Low level rare mobs will often drop common loot). Rare mobs found in instances may also drop an item from the rare table, which will frequently bind on pickup. Rare mobs found in Outland will only drop rare loot, which are all bind on equip. "
TL;DR It wasn't until BC that blizzard began applying logic to the NPCs they are adding.

**Comment by Echinaceax** [comment]
Ah right, understood ...thanks Dranah :)
